finagle-stats-core/src/main/scala/com/twitter/finagle/stats/NonReentrantReadWriteLock.scala

```diff
@@ -23,10 +36,40 @@ private[stats] final class NonReentrantReadWriteLock extends AbstractQueuedSynch
   // we haven't provided an implementation for isHeldExclusively because the
   // docs for AQS advise implementors to provide implementations "as applicable"
   // and our use case doesn't require knowing when the lock is held exclusively.
+  private val SpinMax: Int = if (rwLockSpinWait() == -1) {
+    // Default number of spins for aarch64 machines set to 5 by default. 
+    // See below comment on reason.
+    if (System.getProperty("os.arch") == "aarch_64") {
```

Comment:
ffti: we can omit the `{ }` for single line statement, see: https://docs.scala-lang.org/style/control-structures.html